examples/timeseries/timeseries_classification_from_scratch.py

```diff
@@ -0,0 +1,241 @@
+"""
+Title: Time series classification from scratch
+Author: [hfawaz](https://github.com/hfawaz/)
+Date created: 2020/07/21
+Last modified: 2020/08/17
+Description: Training a time series classifier from scratch on the Coffee dataset from
+the UCR/UEA archive.
+"""
+"""
+## Introduction
+
+This example shows how to do time series classification from scratch, starting from raw
+CSV time series files on disk. We demonstrate the workflow on Coffee dataset from the 
+[UCR/UEA archive](https://www.cs.ucr.edu/%7Eeamonn/time_series_data_2018/).
+
+"""
+
+"""
+## Setup
+
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import accuracy_score
+
+"""
+## Load the data: the Coffee dataset
+
+### Description
+The dataset we are using here is called Coffee.
+The data comes from the UCR archive.
+The dataset contains 28 training instances and another 28 testing instances.
```

Comment:
I had not caught that. In this case I'm afraid the problem setup is inappropriate. The model used here has something like 250,000 floating point parameters. But there are only 28 training data samples. For such a small problem you would use a very small model (e.g [Conv1D(8), Conv1D(num_classes), GlobalMaxPooling()]), not a large model like that. Or even a model that isn't deep-learning based. Deep learning is pattern recognition and as such it requires a reasonable amount of training data.

I would recommend switching to using a problem where each class has at least a few hundreds of samples. Then the problem will be more meaningful.

Reply:
I agree. In fact, I was implementing in this tutorial, a very standard architecture with a very small dataset. 
I will therefore switch to another dataset with few hundred samples per class. 
It should not require big changes to the code. 
Should I open a new pull request or settle for a new commit here ?

Reply:
You could just update this PR when you have a new dataset. For the dataset, the bigger the better. Thank you!